code/controllers/subsystem/explosions.dm

```diff
@@ -207,7 +207,7 @@ SUBSYSTEM_DEF(explosions)
 	for(var/obj/blocking_object in epicenter)
 		if(!blocking_object.density)
 			continue
-			current_exp_block += ( (blocking_object.explosion_block == EXPLOSION_BLOCK_PROC) ? blocking_object.GetExplosionBlock(0) : blocking_object.explosion_block ) //0 is the result of get_dir between two atoms on the same tile.
+		current_exp_block += ( (blocking_object.explosion_block == EXPLOSION_BLOCK_PROC) ? blocking_object.GetExplosionBlock(0) : blocking_object.explosion_block ) //0 is the result of get_dir between two atoms on the same tile.
```

Comment:
Fix the one in admin drawing tools too and report the bug on Spaceman DMM please

Reply:
where's that ?